### PR TITLE
Copy module

### PR DESCRIPTION
**Pull request by xenos92**
### Description

Added description for pandas dataframe.copy() method.

### Type of Change

- Adding a new entry

### Checklist

- [X] All writings are my own.
- [X] My entry follows the Codecademy Docs style guide.
- [X] My changes generate no new warnings.
- [X] I have performed a self-review of my own writing and code.
- [X] I have checked my entry and corrected any misspellings.
- [X] I have made corresponding changes to the documentation if needed.
- [X] I have confirmed my changes are not being pushed from my forked `main` branch.
- [X] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/pandas/concepts/dataframe/terms/copy/copy.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+| Parameter Name | Data Type                | Usage                                                                                                                             |
+| :------------: | ------------------------ | --------------------------------------------------------------------------------------------------------------------------------- |
+|     `deep`     | bool                     | This argument specifies whether the copy should be a deep copy or a shallow copy. If deep=True, then the copy is a deep copy      |
+|    `inplace`   | bool                     | If inplace=False (the default), the copy is made to a new object, and the original object is left unchanged.                      |
+|     `meta`     | python dict              | This argument is used to specify metadata to be attached to the copy                                                              |
+
+## Shallow vs. Deep Copy
+
+Notice that .copy() method has a deep parameter that can dictate the behavior of the resulting dataframe. We can either make a shallow or a deep copy of pandas dataframe.
```
Comment (caupolicandiaz) [inline content/pandas/concepts/dataframe/terms/copy/copy.md:31]:
```suggestion
Note that the designation of the `deep` parameter can have impacts on the resulting copy as well as the original DataFrame.

```

File: content/pandas/concepts/dataframe/terms/copy/copy.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+1. In a shallow copy, the new object points to the same data as the original object, and any changes made to the copy will affect the original object. Changes made to the original dataframe will also be reflected in the shallow copy. By default, .copy() creates a shallow copy. But you can set deep=False to enforce a shallow copy.
```
Comment (caupolicandiaz) [inline content/pandas/concepts/dataframe/terms/copy/copy.md:32]:
```suggestion
- In a shallow copy, the new object points to the same data as the original object, and any changes made to the copy will affect the original object. Changes made to the original dataframe will also be reflected in the shallow copy. By default, the `.copy()` method creates a deep copy. 
```

File: content/pandas/concepts/dataframe/terms/copy/copy.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+
```
Comment (caupolicandiaz) [inline content/pandas/concepts/dataframe/terms/copy/copy.md:33]:
```suggestion
```

File: content/pandas/concepts/dataframe/terms/copy/copy.md [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+2. In a deep copy, a new object is created with a completely new set of data that is identical to the original data. Changes made to the copied object will not affect the original object and vice versa, even if the original data is modified. You can set deep=True to enforce a deep copy.
```
Comment (caupolicandiaz) [inline content/pandas/concepts/dataframe/terms/copy/copy.md:34]:
```suggestion
- In a deep copy, a new object is created with a completely new set of data that is identical to the original data. Changes made to the copied object will not affect the original object and vice versa, even if the original data is modified. 
```

File: content/pandas/concepts/dataframe/terms/copy/copy.md [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+
+## Example:
+
+```py
```
Comment (caupolicandiaz) [inline content/pandas/concepts/dataframe/terms/copy/copy.md:38]: I would recommend that you remove the output comments. Follow the print statement on line 57 with the other two print statements and close the code block. Then have a line of text "The output of the code above will be:" and follow with a ```shell block that just contains the three dataframes.

File: content/pandas/concepts/dataframe/terms/copy/copy.md [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+import pandas as pd
+
+# create a DataFrame
```
Comment (caupolicandiaz) [inline content/pandas/concepts/dataframe/terms/copy/copy.md:41]:
```suggestion
# Create a DataFrame
```

Comment (caupolicandiaz) [reply content/pandas/concepts/dataframe/terms/copy/copy.md:41]: always start comments with a capital

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@xenos92 I've completed my initial review. Please update the entry and tag me when you've finished making changes. Thanks

File: content/pandas/concepts/dataframe/terms/copy/copy.md [hunk 7]
```diff
@@ -0,0 +1,89 @@
...
+---
+
+In Pandas, .copy() is a method that creates a copy of a DataFrame or a Series. This method either returns a fully independent copy of dataframe or a shallow copy (see Shallow vs. Deep Copy section).
+
+## Syntax
+```pseudo
+df_copy = df.copy()
+```
+
+| Parameter Name | Data Type                | Usage                                                                                                                             |
```
[the remark this answers is not in the document]
Comment (caupolicandiaz) [reply content/pandas/concepts/dataframe/terms/copy/copy.md:24]: Yes, please remove them.

**Review by caupolicandiaz — APPROVED** [review]
@xenos92 This is ready for another review.

File: content/pandas/concepts/dataframe/terms/copy/copy.md [hunk 8]
```diff
@@ -0,0 +1,80 @@
...
+df_copy = df.copy()
+```
+The boolean parameter `deep` specifies whether `.copy()` should create a deep or a shallow copy. The default value is `deep=True`, which enforces a deep copy.                                                            
+
+## Shallow vs. Deep Copy
+
+Note that the designation of the `deep` parameter can have impacts on the resulting copy as well as the original DataFrame.
+
+- In a shallow copy, the new object points to the same data as the original object, and any changes made to the copy will affect the original object. Changes made to the original dataframe will also be reflected in the shallow copy. By default, the `.copy()` method creates a deep copy. 
+- In a deep copy, a new object is created with a completely new set of data that is identical to the original data. Changes made to the copied object will not affect the original object and vice versa, even if the original data is modified.
```
Comment (SSwiniarski) [inline content/pandas/concepts/dataframe/terms/copy/copy.md:30]:
```suggestion
- In a deep copy, a new object is created with a completely new set of data that is identical to the original data. Changes made to the copied object will not affect the original object and vice versa. 
```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@xenos92 I did a review and only had one comment.

**Review by SSwiniarski — APPROVED** [review]

**Comment by SSwiniarski** [comment]
@xenos92 Congrats on the new entry. Live link is here: https://www.codecademy.com/resources/docs/pandas/dataframe/copy

You're currently listed as an anonymous contributor. If you want to be credited follow the instructions below to add your GitHub username to your Codecademy profile. If not, there is nothing more you need to do.

image

To add your GitHub username to your Codecademy profile:

1.) Add your GitHub Username to your Codecademy Profile Details, you can get there on your desktop by pressing on your profile picture in the upper right-hand corner and selecting "Profile" in the drop-down box:

image

2.) Then log out and log back into Codecademy using your GitHub login:

image

Note: Your email used for Codecademy will have to be listed in your GitHub profile for the login feature to properly work. If you use the same email for both GitHub and Codecademy already, then you're set.

**Comment by SSwiniarski** [comment]
@yangc95, @KTom101, @caupolicandiaz, any Idea why the entry is still anonymous?

**Comment by KTom101** [comment]
@xenos92 Did you also verify that you have the same email for Codecademy listed under your GitHub profile?

**Comment by yangc95** [comment]
@xenos92 on Codecademy, is your privacy set to this:

Screenshot 2023-04-10 at 4 07 30 PM

**Comment by KTom101** [comment]
@xenos92 It is under your Codecademy profile, the same page you entered your GitHub username. Scroll down to the bottom and the privacy details will be there.

**Comment by KTom101** [comment]
@xenos92 It looks like you're listed now:

image

**Pull request merged**